mdnDocsOverrides.json

```
@@ -467,35 +518,54 @@
   "container-style-queries": [
     "Web/CSS/@container"
   ],
-  "content": [],
+  "content": [
+    "Web/CSS/content"
+  ],
   "content-visibility": [
     "Web/CSS/content-visibility"
   ],
-  "context-fill-stroke": [],
+  "context-fill-stroke": [
+    "Web/SVG/Attribute/fill",
+    "Web/SVG/Attribute/stroke"
+  ],
   "controls-list": [
     "Web/API/HTMLMediaElement/controlsList"
   ],
   "cookie-enabled": [
     "Web/API/Navigator/cookieEnabled"
   ],
-  "cookies": [],
+  "cookies": [
+    "Web/HTTP/Headers/Cookie",
+    "Web/HTTP/Headers/Set-Cookie"
+  ],
```

Let's swap these around, since the Set-Cookie header is the one mentioned in the feature:

```suggestion
  "cookies": [
      "Web/HTTP/Headers/Set-Cookie",
    "Web/HTTP/Headers/Cookie"
  ],
```